### DISCUSSION TITLE
Project very very slow on windows11

### DISCUSSION THREAD

**Discussion by sfili81** [dunglas/symfony-docker#576] [Q&A] [answered]
I'm using dunglas/symfony-docker just clone it and followed the instruction inside the read.me to install and use it
The project is very slow, for example:
![Screenshot 2024-02-26 at 15-47-37 Symfony Profiler](https://github.com/dunglas/symfony-docker/assets/7912591/7ff25f93-514c-4de1-8c30-29a40445e090)

![Screenshot 2024-02-26 at 15-43-09 Error Maximum execution time of 30 seconds exceeded (500 Internal Server Error)](https://github.com/dunglas/symfony-docker/assets/7912591/858b5a7f-7a59-43da-8a6f-565e92e7c26e)

How can i speed up this docker?

**Comment by maxhelias** [comment, marked as answer]
See : https://github.com/dunglas/symfony-docker/blob/e7d8ef74864ea06bac3d819787023f7e9cb70dbb/compose.override.yaml#L11-L13

I couldn't suggest anything better, the performance issues are not related to the project but to the docker file system on the different OS.
Window is known for having the worst performance and we can't do anything about it on our own.

**Reply by sfili81** [reply]
Thank you, i'll see it but i don't understadn so much i enable this line but i get error : cannot remove app vendor

**Reply by maxhelias** [reply]
Uncommenting this line will create a volume used by Docker containers instead of sharing the directory with your host machine, which for Mac and Window uses a different filesystem. 
So Docker doesn't have to do any conversion processing when the files are in sync because they won't be sync on your machine.

**Comment by 7-zete-7** [comment]
Try to move your project to WSL's directory.

Stored project on Windows' folders will be adapted by WSL's filesystem converter to be used by Docker. It increases CPU usage and makes project too lagy.

**Reply by sfili81** [reply]
I'll read something about it but i don't know how to move my project inside WSL's directory

**Reply by 7-zete-7** [reply]
On stable WSL with `Ubuntu` distro:
```shell
cp "<your_project_dir_path>" "\\wsl$\Ubuntu\home\user\projects\awesome-project"
```

On unstable WSL with `Ubuntu` distro:
```shell
cp "<your_project_dir_path>" "\\wsl.localhost\Ubuntu\home\user\projects\awesome-project"
```

In both examples used `user` WSL user. If your username is different — use own username.

Example with `Debian` distro and `awesome-user` username:
```shell
cp "D:\PhpProjects\awesome-project" "\\wsl$\Debian\home\awesome-user\projects\awesome-project"
```

Any editor or IDE can work with projects in WSL. Just open project by the new location.
